pkg/terminal/color/stack.go

```diff
@@ -55,15 +55,33 @@ func (s *Stack) ColorEnd(b *strings.Builder) {
 	}
 }
 
-func (s *Stack) ColorStartBytes(b *bytes.Buffer, code Code) {
+func (s *Stack) StartInBytes(b *bytes.Buffer, code Code) {
```

Comment:
`bytes.Buffer` and `strings.Builder` both implement `io.Writer`, so you don't really need two versions of the `Start` and `End` methods.
```suggestion
func (s *Stack) Start(w io.Writer, code Code) {
```

Reply:
sweet!